CMSgov/design-system
# [WNMGDS-970] form label typescript conversion

## Summary
Converting the FormLabel component to typescript

## How to test
http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-970/form-label-typescript-conversion
Navigate to the FormLabel component and confirm there aren't any console errors.
Also confirm all expected props show up in documentation
